Core/src/org/sleuthkit/autopsy/casemodule/Case.java

```diff
@@ -223,8 +205,16 @@ private Case(String name, String number, String examiner, String configFilePath,
         this.caseType = type;
         this.db = db;
         this.services = new Services(db);
-        this.eventSubscriber = new EventSubscriber();
-        this.messenger = new Messenger(this.name, eventPublisher);
+        if (CaseType.MULTI_USER_CASE == this.caseType) {
+            try {
+                eventPublisher.openRemoteEventChannel(name);
```

Comment:
Would it be better to do this in an init() after the case is open? That way we're not subscribed for incoming messages until we're fully constructed. And, if this fails, the user is unable to edit anything with the case, right? e.g. they can't add a data source or anything... right?